distribution/lib/Standard/Database/0.0.0-dev/src/Connection/Postgres.enso

```diff
@@ -30,7 +30,14 @@ type Postgres
        - use_ssl: Whether to use SSL (defaults to `SSL_Mode.Prefer`).
        - client_cert: The client certificate to use or `Nothing` if not needed.
     @credentials Credentials.default_widget
-    Server (host:Text=default_postgres_host) (port:Integer=default_postgres_port) (database:Text=default_postgres_database) (schema:Text="") (credentials:(Credentials|Nothing)=Nothing) (use_ssl:SSL_Mode=SSL_Mode.Prefer) (client_cert:(Client_Certificate|Nothing)=Nothing)
+    Server
+        host : Text = default_postgres_host
+        port : Integer = default_postgres_port
+        database : Text = default_postgres_database
+        schema : Text = ""
+        credentials : Credentials|Nothing = Nothing
+        use_ssl : SSL_Mode = SSL_Mode.Prefer
+        client_cert : Client_Certificate|Nothing = Nothing
```

Comment:
Here's an example of the new *constructor argument-definition line* syntax.

Reply:
This may be [related documentation](https://github.com/enso-org/enso/blob/171a95f17acdae71f72c6b5ae5e46629d2aad4e3/docs/types/types.md?plain=1#L17) to update.